### ISSUE TITLE
Lifecycle notifications in v1.8.0 do not work for V1 function apps

### ISSUE THREAD

**Issue by yuka1984** [Azure/azure-functions-durable-extension#670]
Hi. I tried version 1.8.0 on V1 Functions.

But get error and not worked.

This error code is this.

```
System.ArgumentNullException:
   at Microsoft.Azure.WebJobs.Extensions.DurableTask.EventGridLifeCycleNotificationHelper..ctor (Microsoft.Azure.WebJobs.Extensions.DurableTask, Version=1.0.0.0, Culture=neutral, PublicKeyToken=014045d636e89289)
   at Microsoft.Azure.WebJobs.Extensions.DurableTask.DurableTaskExtension.CreateLifeCycleNotificationHelper (Microsoft.Azure.WebJobs.Extensions.DurableTask, Version=1.0.0.0, Culture=neutral, PublicKeyToken=014045d636e89289)
   at Microsoft.Azure.WebJobs.Extensions.DurableTask.DurableTaskExtension.InitializeForFunctionsV1 (Microsoft.Azure.WebJobs.Extensions.DurableTask, Version=1.0.0.0, Culture=neutral, PublicKeyToken=014045d636e89289)
   at Microsoft.Azure.WebJobs.Extensions.DurableTask.DurableTaskExtension.Microsoft.Azure.WebJobs.Host.Config.IExtensionConfigProvider.Initialize (Microsoft.Azure.WebJobs.Extensions.DurableTask, Version=1.0.0.0, Culture=neutral, PublicKeyToken=014045d636e89289)
   at Microsoft.Azure.WebJobs.Host.Executors.JobHostConfigurationExtensions.InvokeExtensionConfigProviders (Microsoft.Azure.WebJobs.Host, Version=2.4.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35)
   at Microsoft.Azure.WebJobs.Host.Executors.JobHostConfigurationExtensions.CreateStaticServices (Microsoft.Azure.WebJobs.Host, Version=2.4.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35)
   at Microsoft.Azure.WebJobs.JobHost.InitializeServices (Microsoft.Azure.WebJobs.Host, Version=2.4.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35)
   at Microsoft.Azure.WebJobs.Script.Utility.CreateMetadataProvider (Microsoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=nullMicrosoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: C:\projects\azure-webjobs-sdk-script\src\WebJobs.Script\Utility.csMicrosoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: 362)
   at Microsoft.Azure.WebJobs.Script.ScriptHost.LoadBindingExtensions (Microsoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=nullMicrosoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: C:\projects\azure-webjobs-sdk-script\src\WebJobs.Script\Host\ScriptHost.csMicrosoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: 966)
   at Microsoft.Azure.WebJobs.Script.ScriptHost.Initialize (Microsoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=nullMicrosoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: C:\projects\azure-webjobs-sdk-script\src\WebJobs.Script\Host\ScriptHost.csMicrosoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: 299)
   at Microsoft.Azure.WebJobs.Script.ScriptHostManager.RunAndBlock (Microsoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=nullMicrosoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: C:\projects\azure-webjobs-sdk-script\src\WebJobs.Script\Host\ScriptHostManager.csMicrosoft.Azure.WebJobs.Script, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: 178)
```

This error code is this.

https://github.com/Azure/azure-functions-durable-extension/blob/dev/src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs#L200

I creating this error's PR.

**Comment by TsuyoshiUshio** [comment]
Maybe we need CI for both on .Net Standard and Framework.

**Comment by cgillum** [comment]
Our CI runs both .NET Standard 2.0 (v2) and .NET Framework (v1) tests, but we don't run all tests for .NET Framework - only a subset. This is because v1 tests run much slower and we wanted to reduce CI time. Unfortunately we did not have any event grid notification tests running in the .NET Framework (v1) tests. That is why this problem was not caught by the CI. We will need to fix this by enabling some event grid tests to run in the .NET Framework tests.

**Comment by cgillum** [comment]
Actually, I just noticed that our functional tests do not execute this code path the same way that a normal v1 function app does. I will see if I can refactor the functional tests to better verify whether v1 is working correctly.

**Comment by cgillum** [comment]
This has been fixed in the [v1.8.1 release](https://github.com/Azure/azure-functions-durable-extension/releases/tag/v1.8.1).
